lib/plugins/game.js

```diff
@@ -67,7 +67,6 @@ function inject (bot, options) {
 
     bot.emit('login')
     bot.emit('game')
-    bot._client.write('held_item_slot', { slotId: 0 })
```

Comment:
why

Reply:
Vanilla client only sends the server bound held item packet when recieving a client bound held item packet